doc/devel/documenting_mpl.rst

```diff
@@ -19,6 +24,31 @@ The requirements are as follows (https://github.com/matplotlib/matplotlib/blob/m
 6. pillow
 7. graphviz
 
+.. note::
+
+  * You'll need a minimal working latex distribution for many examples to run.
+  * `Graphviz <http://graphviz.readthedocs.io/en/latest/manual.html>`_ is not pip-installable so you need
+    to install this on your own.
+
+General structure
+-----------------
+
+All documentation is built from the :file:`doc/` directory. This folder contains both
```

Comment:
One case of "directory" and many of "folder"; I think we should be consistent about it (preferring directory, myself.)